server/events/pending_plan_finder.go

```diff
@@ -58,8 +58,8 @@ func (p *DefaultPendingPlanFinder) findWithAbsPaths(pullDir string) ([]PendingPl
 		lsCmd.Dir = repoDir
 		lsOut, err := lsCmd.CombinedOutput()
 		if err != nil {
-			return nil, nil, errors.Wrapf(err, "running git ls-files . "+
-				"--others: %s", string(lsOut))
+			return nil, nil, errors.Wrapf(err, "Error running 'git ls-files . "+
```

Comment:
https://github.com/runatlantis/atlantis/blob/main/CONTRIBUTING.md#errors indicates this should be `running` instead of `Error running`. However given the screenshot on the previous page, having `Error` at the front seems to make more sense. @nitrocode as a takeaway we should either update the doc, or maybe have the output in the atlantis comment include the `Error`?

Reply:
I'm going to resolve my own comment because it shouldn't block this PR, just something for us to look into.

Reply:
I unresolved per @jamengual; we should keep the convention and use `errors.Wrapf(err, "running ...`

Reply:
Ok, so I refactor it to meet the requirements